deps/rabbit/src/rabbit_queue_type.erl

```diff
@@ -765,3 +766,13 @@ known_queue_type_names() ->
     {QueueTypes, _} = lists:unzip(Registered),
     QTypeBins = lists:map(fun(X) -> atom_to_binary(X) end, QueueTypes),
     ?KNOWN_QUEUE_TYPES ++ QTypeBins.
+
+check_vhost_queue_limit(Q) ->
+    #resource{name = QueueName} = amqqueue:get_name(Q),
+    VHost = amqqueue:get_vhost(Q),
+    case rabbit_vhost_limit:is_over_queue_limit(VHost) of
+        false         -> ok;
```

Comment:
redundant whitespace, there is no need to arbitrarily line up things like `->` just use a newline instead. Same for the clause below.

Reply:
Agree, I just moved the function from rabbit_channel, and didnt want to change it but I am happy to make it look better.